codice/ddf-ui
# [3.6.x] Fixes styling issues with System Usage Modal

Rewrites System Usage Modal to make it consistent with theme
![Screen Shot 2020-08-20 at 15 57 25](https://user-images.githubusercontent.com/9013407/90833915-dcf70180-e2fd-11ea-84c9-61c5b3c3fd32.png)

To test: 
Configure `ddf.platform.ui.config` System Usage title/Body, use html for the body if you like.
Verify system usage modal appears as expected. **Verify you can't click outside the modal to close.**
**Note**: There is a weird visual effect when loading in the page- I don't think this is a big deal but it is a little strange.
